src/dom/rect.js

```diff
@@ -82,10 +84,9 @@ export default class Rect {
 				 * @error rect-source-not-in-dom
 				 * @param {String} source The source of the Rect instance.
 				 */
-				log.warn(
+				console.warn( attachLinkToDocumentation(
 					'rect-source-not-in-dom: The source of this rect does not belong to any rendered DOM tree.',
-					{ source }
-				);
+				), { source } );
```

Comment:
This should be displayed only in the debug mode, this error tells nothing to end users.

Reply:
Maybe the error code/message could be better? Because I think that the warning should be displayed at least.